docs/documenting.md

```diff
@@ -20,7 +20,7 @@ For larger changes, we suggest that you clone the website repository. This way,
 
 The easiest way to get started with development is to use Visual Studio Code with devcontainers, in the same way as for working on Home Assistant Core development. Have a look at the [Development Environment](https://developers.home-assistant.io/docs/development_environment) page for instructions.
 
-To review the changes, open the command palette and choose ´Tasks: Run Task´ and then ´Preview´.
+To review the changes, open the command palette and choose ´Tasks: Run Task´ and then ´Run Home Assistant Core´.
```

Comment:
This section talks about how to run the user documentation site preview task in vscode. That's still called `Preview`.

https://github.com/home-assistant/home-assistant.io/blob/6f754550858576fa945dcf1a6968066b8c273b56/.vscode/tasks.json#L14

Reply:
You are right, I was running core but looking at the wrong doc, sorry about that.